app/models/ems_folder.rb

```diff
@@ -150,6 +150,18 @@ def register_host(host)
     end
   end
 
+  def move_into_folder(vm)
```

Comment:
Why is this implemented at the base ems_folder level instead of in the vmware provider?

Reply:
I would be ok if this was implemented at the base as "raise NotImplementedError" and in vmware as doing the concrete work.

Reply:
There isn't a folder class in the VMware provider currently

Reply:
I'm good with adding one though